x-pack/plugins/case/common/api/connectors/index.ts

```diff
@@ -7,25 +7,34 @@
 
 import * as rt from 'io-ts';
 
+import { ActionResult, ActionType } from '../../../../actions/common';
 import { JiraFieldsRT } from './jira';
 import { ResilientFieldsRT } from './resilient';
-import { ServiceNowFieldsRT } from './servicenow';
+import { ServiceNowITSMFieldsRT } from './servicenow_itsm';
+import { ServiceNowSIRFieldsRT } from './servicenow_sir';
 
 export * from './jira';
-export * from './servicenow';
+export * from './servicenow_itsm';
+export * from './servicenow_sir';
 export * from './resilient';
 export * from './mappings';
 
+export type ActionConnector = ActionResult;
+export type ActionTypeConnector = ActionType;
+
 export const ConnectorFieldsRt = rt.union([
   JiraFieldsRT,
   ResilientFieldsRT,
-  ServiceNowFieldsRT,
+  ServiceNowITSMFieldsRT,
+  ServiceNowSIRFieldsRT,
```

Comment:
I am wondering if we just call it ServiceNowSecOps instead of the ServiceNowSIR to avoid a misunderstanding between what we call it internally and externally.